DMX.Core.Api.Tests.Acceptance/APIs/LabCommands/LabCommandApiTests.cs

```diff
@@ -36,6 +36,7 @@ private static LabCommand UpdateRandomLabCommand(LabCommand inputLabCommand)
                 .OnProperty(labCommand => labCommand.Id).Use(inputLabCommand.Id)
                 .OnProperty(labCommand => labCommand.CreatedDate).Use(inputLabCommand.CreatedDate)
                 .OnProperty(labCommand => labCommand.UpdatedDate).Use(now);
+            //.OnType<DateTimeOffset>().Use(GetRandomDateTime());

```

Comment:
Is this supposed to be a comment?

Reply:
Holy ..
Where did that come from?!

Thanks.